build/build-image/Dockerfile

```diff
@@ -16,6 +16,13 @@
 
 FROM debian:bullseye
 
+ARG USER_ID
+ARG GROUP_ID
+RUN groupadd -g ${GROUP_ID:-1338} agones-dev && \
+    useradd -l -u ${USER_ID:-1337} -g agones-dev -s /bin/bash agones-dev && \
+    install -d -m 0755 -o agones-dev -g agones-dev /home/agones-dev && \
+    mkdir /home/agones-dev/.cache && mkdir /home/agones-dev/.config
```

Comment:
Curious why you need to create these directories? I take it they don't get created by default by the tools that need them?